src/styles/_catalog.scss

```diff
@@ -1,22 +1,29 @@
+$grid-width: 200px;
+$layout-breakpoint-small: 488px;
+$layout-breakpoint-medium: 768px;
+$layout-breakpoint-large: 1024px;
+
+@mixin columns($width, $cols) {
+  @media (min-width: $width) {
+    grid-template-columns: repeat($cols, $grid-width);
+  }
+}
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: $grid-width;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @include columns($layout-breakpoint-small, 2);
+  @include columns($layout-breakpoint-medium, 3);
+  @include columns($layout-breakpoint-large, 4);
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
+    box-sizing: border-box;
     width: 200px;
     height: 300px;
```

Comment:
text of the card is not centered, you can use grid for that)
![image](https://user-images.githubusercontent.com/37161315/170269780-06f52482-3ea4-45ec-9286-85a9f3f9b1fa.png)

Reply:
Also removed box-sizing: border box from here, it was in main scss. moved it to normalize.scss.